docs/source/index.rst

```diff
@@ -48,7 +48,12 @@ on command-line. The following configuration is useful to setup.
 Basic Configuration
 --------------------
 
-Flytectl configuration
+Flytectl configuration. 
+For full list of configurable options available can be found by running ``flytectl --help`` and can be alternately be found `here <https://docs.flyte.org/projects/flytectl/en/stable/gen/flytectl.html#synopsis>`__
```

Comment:
where is this `flytectl.html` in the table of contents? I can't seem to find it from the flytectl home page: https://docs.flyte.org/projects/flytectl/en/stable/

Reply:
Done . thanks for pointing out. Have added to index.rst